core/src/edu/asu/c3simulator/screens/CurrentProducts.java

```diff
@@ -0,0 +1,202 @@
+ * @author Reigel, Justin
+ * 
+ */
+public class CurrentProducts implements Screen
+{
+	private static final String ADVISOR_TEXT = "This is the Current Products screen";
+	private static final int DESIGN_WIDTH = 1280;
+	private static final int DESIGN_HEIGHT = 720;
+	private Stage stage;
+	private Skin skin;
+	private Game game;
+	// private CornerAdvisor advisor;
+	private Table table;
+	private BitmapFont font = new BitmapFont(
+			Gdx.files.internal("fonts/arial32_superSample.fnt"));
+	private ManagePanel manage;
+
+	public CurrentProducts(Game game)
+	{
+		this.game = game;
+
+		Viewport stageViewport = new StretchViewport(DESIGN_WIDTH,
+				DESIGN_HEIGHT);
+		this.stage = new Stage(stageViewport);
+		this.skin = new Skin(Gdx.files.internal("skins/default/uiskin.json"));
+		// TODO: uncomment advisor when advisor is implemented in master_testing
+		// or higher.
+
+		// advisor = new CornerAdvisor(ADVISOR_TEXT);
+		// creates and adds manage Panel to the current screen
+		// manage = new ManagePanel(game);
+
+		table = new Table();
+		table.setSize(900, 500);
+		table.add(ProductPane()).fill().expand();
+		table.setPosition(300, 50);
+		table.add(ProductInfo()).size(300, 500);
+
+		float padding = 0.01f * DESIGN_HEIGHT;
+		// float advisorLeft = DESIGN_WIDTH - advisor.getPrefWidth() - padding;
+		// float advisorBottom = DESIGN_HEIGHT - advisor.getPrefHeight() -
+		// padding;
+		// advisor.setPosition(advisorLeft, advisorBottom);
+
+		HomeButton home = new HomeButton(game);
+
+		stage.addActor(home);
+		// stage.addActor(advisor);
+		stage.addActor(table);
+	}
+
+	/**
+	 * Initializes management panel. Called during initialization to avoid
+	 * infinite loops with AllManagementScreens.
+	 */
+	public void initialize()
+	{
+		manage = new ManagePanel(game, stage);
+	}
+
+	/**
+	 * Creates a table of the current product icons in a panel that the can be
+	 * scrolled through.
+	 * 
+	 * @return Table
+	 */
+	private Actor ProductPane()
+	{
+		TextButton text = new TextButton("Designs", skin);
+		text.setDisabled(true);
+		text.setColor(0.5f, 0.5f, 0.5f, 1.0f);
+
+		Table productTable = new Table();
+		// TODO: replace with real products and product icons.
+		productTable.add(new HomeButton(game));
+		productTable.add(new HomeButton(game));
+		productTable.add(new HomeButton(game));
+		productTable.row();
+		productTable.add(new HomeButton(game));
+		productTable.add(new HomeButton(game));
+		productTable.row();
+		productTable.add(new HomeButton(game));
+		productTable.row();
+		productTable.add(new HomeButton(game));
+
+		ScrollPane chosenTasksScroll = new ScrollPane(productTable, skin);
+
+		Table leftSection = new Table();
+		leftSection.debug();
+		leftSection.add(text).fillX().align(Align.center);
+		leftSection.row();
+		leftSection.add(chosenTasksScroll).expand().fill().center();
+		return leftSection;
+	}
+
+	/**
+	 * Creates a table with detailed information about the current products in a
+	 * panel that the can be scrolled through.
```

Comment:
The documentation says it uses information about the current product, but the functionality does not reflect this. Will this panel be updated, or recreated when the current product changes? (or a new product becomes the current product)